atd-etl/cris_import/cris_import.py

```diff
@@ -75,7 +76,19 @@ def main():
 
     SFTP_ENDPOINT_SSH_PRIVATE_KEY = secrets["sftp_endpoint_private_key"]
 
-    zip_location = download_archives()
+    local_mode = False
+    if bool(glob.glob('/app/development_extracts/*.zip')):
```

Comment:
I tested this with two different extracts that I grabbed from the sftp server, and I saw the script pick them up with no issue. ✨
<img width="843" alt="Screenshot 2023-09-29 at 12 33 07 PM" src="https://github.com/cityofaustin/atd-vz-data/assets/37249039/4e3b4a4e-8fd4-49df-a360-0cd87b4ac599">